### ISSUE TITLE
Feature broadcast build success message appears multiple times

### ISSUE THREAD

**Issue by ddgenome** [atomisthq/spring-team-handlers#35]
<img width="1005" alt="multiple-build-messages" src="https://user-images.githubusercontent.com/57881/26843396-b47caea0-4ab5-11e7-8cb4-0183850b8658.png">

**Comment by ddgenome** [comment]
@cdupuis does this commit ae8ea8efcc5e0243f511b6d2e1cdb778f5351ced address this issue?

**Comment by cdupuis** [comment]
I don't think it actually does. This seems to be a similar issue somewhere else.
